### PR TITLE
homepage demo update

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Last night I noticed that the demos on the landing page don't reflect the latest "gen ai boom" so I added the llama2 ChatInterface and a Stable Diffusion demo.

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

**Comment by aliabid94** [comment]
Awesome! I do think the order of these should introduce Interface, then Blocks, then ChatInterface. 

I also think we can drop or swap out the echocardiogram one too possibly. Also the Stable Diffusion one's layout looks weird, lemme see if I can fix hysts space

**Comment by freddyaboulton** [comment]
> I do think the order of these should introduce Interface, then Blocks, then ChatInterface.

Completely agreed!

**Comment by aliabid94** [comment]
@hysts https://huggingface.co/spaces/hysts/SD-XL/discussions/15

File: js/_website/src/lib/components/DemosLanding.svelte [hunk 1]
```diff
@@ -1,18 +1,21 @@
 <script lang="ts">
-	import { q_a, img, sketch } from "../assets/demo_code";
+	import { stable_diffusion, img, sketch, chat } from "../assets/demo_code";
```
Comment (freddyaboulton) [inline js/_website/src/lib/components/DemosLanding.svelte:2]: Much easier to update this than the raw html. Nice work @aliabd @pngwn

**Review by aliabd — APPROVED** [review]
Code looks good! I agree with @aliabid94 that we should probably kill Echocardiogram

**Review by pngwn — APPROVED** [review]
Looks good! Thanks for taking care of this @freddyaboulton!

**Comment by freddyaboulton** [comment]
Thank you all for the input!!

**Pull request merged**